sdk/src/network-client.ts

```diff
@@ -632,40 +700,56 @@ class AleoNetworkClient {
    */
   async getProgramMappingPlaintext(programId: string, mappingName: string, key: string | Plaintext): Promise<Plaintext> {
     try {
-      const keyString = key instanceof Plaintext ? key.toString() : key;
-      const value = await this.fetchRaw("/program/" + programId + "/mapping/" + mappingName + "/" + keyString);
-      return Plaintext.fromString(JSON.parse(value));
+      const value = await this.getProgramMappingValue(programId, mappingName, key);
+      return Plaintext.fromString(value);
     } catch (error) {
-      throw new Error("Failed to fetch mapping value." + error);
+      throw new Error(`${error}`);
     }
   }
 
   /**
    * Returns the latest state/merkle root of the Aleo blockchain.
    *
+   * @param {number} height - The height for which the state root is fetched, if blank then the latest state root will be returned.
+   * 
```

Comment:
```suggestion
   * 
```

There's no height parameter accepted in this function.